src/styles/Form.module.css

```diff
@@ -0,0 +1,69 @@
+:root {
+  --grey-one: #888;
+  --light: #fff;
+  --grey-dark: #e8e8e8;
+  --dark-one: #121212;
+  --blue-sky: #0290ff;
+}
+
+.formContainer {
+  width: 90%;
+  margin: 0 auto;
+}
+
+hr {
+  background: var(--grey-dark);
+  height: 3px;
+  border: none;
+}
```

Comment:
- [ ] As I mentioned already in my above two comments, you need to follow the styles exactly given in the zeplin's design, I think right now you need to add the padding on top and bottom, because if you look into the original design there is enough space given before and after the horizontal rule. In your design because of having very little space, the design does not look that good, Please add the spacing then it will look fine.

## Requirement
![Screenshot from 2023-08-25 19-39-00](https://github.com/kambereBr/bookstore/assets/66476329/0b10d308-79f2-4892-8ba9-ef2ab2dd4849)

### Design Comparison
<table>
<tr>
<th>Zeplin Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-08-26 17-01-20](https://github.com/kambereBr/bookstore/assets/66476329/da57914a-d772-4b59-87bc-2212bdeb4a36)


</td>
<td>

![Screenshot from 2023-08-26 17-01-29](https://github.com/kambereBr/bookstore/assets/66476329/9901bc29-c47b-4198-9867-adce50bd39e0)


</td>
</tr>
</table>